libwallet/instantswap.go

```diff
@@ -126,47 +126,57 @@ func (mgr *AssetsManager) StartScheduler(ctx context.Context, params instantswap
 
 		market := values.NewMarket(fromCur, toCur)
 		source := mgr.RateSource.Name()
-		ticker := mgr.RateSource.GetTicker(market, false) // TODO: (@ukane-philemon) Can we proceed if there is no rate information from rate source? I think we should.
+		ticker := mgr.RateSource.GetTicker(market, false)
 		if ticker == nil {
-			return errors.E(op, fmt.Errorf("unable to get market rate from %s", source))
+			log.Errorf("unable to get market(%s) rate from %s.", market, source)
+			log.Infof("Proceeding without checking market rate deviation...")
+		} else {
+			exchangeServerRate := res.ExchangeRate // estimated receivable value for libwallet.DefaultRateRequestAmount (1)
```

Comment:
why is this using `res.ExchangeRate` instead of `res.EstimatedAmount `

Reply:
The current calculation checks for market price deviation using the rate. before now the estimated amount was used which was incorrectly used and also as the estimate might change based on some factors like the final invoiced amount.